openjdk/src/test/java/org/conscrypt/DuckTypedPSKKeyManagerTest.java

```diff
@@ -75,19 +84,22 @@ public void testDuckTypingFailsWhenOneMethodReturnTypeIncompatible() throws Exce
         } catch (NoSuchMethodException expected) {}
```

Comment:
`KeyManagerOfferingAllPSKKeyManagerMethodsWithIncompatibleReturnTypes` must win some kind of prize :)